pkg/client/mount.go

```diff
@@ -258,25 +265,34 @@ func mount(ctxt context.Context, source, dest string, options []string, fs strin
 	dev, err := deviceFromMount(ctxt, source)
 
 	// If we couldn't resolve, then we're probably working with the device already
```

Comment:
This comment is not relevant anymore; also related to the previous comment in deviceFormMount() function above.

Reply:
comment removed